lib/handbook/addon/docs/visual-style/template.md

```diff
@@ -1,5 +1,98 @@
+
+
+### Take steps to make website functional in mobile devices
+- Check [responsiveness](http://www.smashingmagazine.com/2011/01/12/guidelines-for-responsive-web-design/) in every page.
+- Make Header texts proportionally smaller in smaller devices, no need for an H1 in a phone.
+- Make text that does not fit constrained widths end with ...
+- Make different versions for mobile buttons that have less text. - Make sure all images are responsive.
+- Check show/hide relationships in mobile devices, often it's unnecessary.
+- Check empty vertical space in rows.
+- Check that horizontally listed items stack properly in narrow view.
+- Check that tables respond properly to narrow views, scrolling if necessary.
+- Make sure touch and event based actions either work well or have fallbacks in mobile devices.
+
+### Take steps to make sure the website is functional in very large devices
+Where possible, let the user utilize the large workspace that they have by providing meaningful expansion of the space:
+
+- Expand tables, grids, and stacked components that have uniform items (e.g. galleries).
+- Do not expand Text! Instead fit several columns.
+- Remove show/hide options to show everything as space allows, that reduces an extra step.
+- Stack similar content horizontally as columns.
+- Center content if not utilizing the entire width.
+
+### Improve the form input and toggle process where possible
+Follow these guidelines when making forms or add elements such as drop downs.
+
+- If your dropdown has few elements, consider another input or toggle method, like a button group or radio buttons.
+- Make input elements large and text visible.
+- Uniformity in input elements is important, stick to themes or frameworks; if you are adding custom css to established framework css for inputs you are most likely doing it wrong.
+- Do not remove the visual separation of focus on input elements, but make it fit to your general theme.
+- Most of the time select and checkbox items are too small, consider increasing their size or generally make sure they fill your font's line height.
+- Make sure tab goes to the next item if there is a form with multiple items.
+- If using JS for form inputs, bind Enter key.
+- Don't have a "Clear Form" button.
+- Avoid using too many borders in form structures. Alignment is a way to denote edges as well and inputs already have borders.
+- Use a good combination of placeholder and label for inputs, often one is enough, both are unnecessary. Don't repeat the label in the placeholder. Label is better for cross browser use.
+- Tie your labels to your form elements with "for" so that when users click on the label the input gets selection.
+
+### Respect history
+If you are using a single page approach where page loads are not required for significant events, remember that users consider significant layout changes to denote page changes and will want to press back to go back to the previous state.
```

Comment:
Doesn't make sense to specify this in an ember app, it should happen automatically.